account_credit_control/report/credit_control_summary.html.mako

```diff
@@ -205,6 +214,23 @@ tr.line {
         <td class="amount">${line.balance_due}</td>
         <td class="amount">${line.currency_id.name or comm.company_id.currency_id.name}</td>
       </tr>
+      <%
+        currency = line.currency_id.name or comm.company_id.currency_id.name
+        if currency in balance_due_per_currency:
+            balance_due_per_currency[currency] += line.balance_due
+        else:
+            balance_due_per_currency[currency] = line.balance_due
+      %>
+%endfor
+      </table>
+      <br/>
+      <table class="list_table" style="width: 100%;">
+%for currency in balance_due_per_currency.keys():
```

Comment:
Isn't it possible to use?

    %for currency, balance_due in balance_due_per_currency.iteritems():